src/box/lua/config/applier/console.lua

```diff
@@ -3,6 +3,8 @@ local console = require('console')
 local log = require('internal.config.utils.log')
 local instance_config = require('internal.config.instance_config')
 
+local old_socket = nil
```

Comment:
Nit: It is not necessarily 'old' (that sounds like 'non-actual'). Just 'last', 'saved', 'previous', 'existing'.

Reply:
Please, leave comments open: it is easier to look over them on the next review iteration.

Reply:
corrected